include/control.h

```diff
@@ -40,15 +40,42 @@ enum class Verbosity : int8_t {
 	High,          //   yes   |    yes       |
 };
 
+struct CommandLineArguments {
+	bool printconf;
+	bool noprimaryconf;
+	bool nolocalconf;
+	bool fullscreen;
+	bool list_glshaders;
+	bool version;
+	bool help;
+	bool eraseconf;
+	bool erasemapper;
+	bool noconsole;
+	bool startmapper;
+	bool exit;
+	std::string opencaptures;
+	std::string working_dir;
+	std::string lang;
+	std::string machine;
+	std::vector<std::string> conf;
+	std::vector<std::string> set;
+	std::optional<std::vector<std::string>> editconf;
+	std::optional<int> socket;
```

Comment:
It's a good idea to keep using the command line switch names here for clarity 👍🏻